wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/CheEnvironmentEngine.java

```diff
@@ -940,7 +940,9 @@ private Instance startInstance(boolean recover,
                         }
                     } catch (NotFoundException e) {
                         try {
-                            machineLogger.writeLine("Failed to boot machine from snapshot: snapshot not found. " +
+                            machineLogger.writeLine("Failed to boot machine " + machine.getConfig().getName() +
```

Comment:
String.format("Workspace id=%s. Failed to boot machine %s, reason: snapshot not found. "Machine will be created from origin source.", 
           machine.getWorkspaceId(),
           machine.getConfig().getName())